server/index.js

```diff
@@ -10,7 +10,7 @@ app.get('/api', (req, res) => {
     if (err) throw err;
     res.send(result);
   });
-  db.end();
```

Comment:
Because this was outside of the query callback, it was most likely hitting `db.end()` before the query was even completed! That would be my best guess anyway.